services/slam/fake/slam_test.go

```diff
@@ -104,6 +140,40 @@ func TestFakeSLAMGetMap(t *testing.T) {
 	})
 }
 
+func TestFakeSLAMGetInternalStateStream(t *testing.T) {
+	slamSvc := &SLAM{Name: "test", logger: golog.NewTestLogger(t)}
+
+	data := getDataFromStream(t, slamSvc.GetInternalStateStream, slamSvc.Name)
+	test.That(t, len(data), test.ShouldBeGreaterThan, 0)
+
+	data2 := getDataFromStream(t, slamSvc.GetInternalStateStream, slamSvc.Name)
+	test.That(t, data, test.ShouldResemble, data2)
+}
+
+func TestFakeSLAMGetPointMapStream(t *testing.T) {
+	slamSvc := &SLAM{Name: "test", logger: golog.NewTestLogger(t)}
+
+	data := getDataFromStream(t, slamSvc.GetPointCloudMapStream, slamSvc.Name)
+	test.That(t, len(data), test.ShouldBeGreaterThan, 0)
```

Comment:
[q] do we want to test that a full Pointcloud is formed and/or equals the set of bytes in the artifact path? That second one is tricky as we don't necessary have the artifact path being called (although it can be predicted)

Reply:
Added